### ISSUE TITLE
[TRI-1149] bug: Cannot get eslint-plugin working on a fresh Next.js project

### ISSUE THREAD

**Issue by ericallam** [triggerdotdev/trigger.dev#391]
Here's the reproduction project:

https://github.com/ericallam/my-app

As you can see I've setup the eslint-plugin, but for some reason `npm run lint` isn't finding any errors in this code:

```ts
import { Job, eventTrigger } from "@trigger.dev/sdk";
import { client } from "@/trigger";

// your first job
new Job(client, {
  id: "example-job",
  name: "Example Job",
  version: "0.0.1",
  trigger: eventTrigger({
    name: "example.event",
  }),
  run: async (payload, io, ctx) => {
    await io.runTask("example.task", { name: "Task 1" }, async () => {});
    await io.runTask("example.task", { name: "Task 2" }, async () => {});
  },
});
```

cc: @ologbonowiwi

<sub>[TRI-1149](https://linear.app/triggerdotdev/issue/TRI-1149/bug-cannot-get-eslint-plugin-working-on-a-fresh-nextjs-project)</sub>

**Comment by ologbonowiwi** [comment]
Likely, the issue is on [/packages/eslint-plugin/lib/rules/no-duplicated-task-keys.js:84](https://github.com/triggerdotdev/trigger.dev/blob/main/packages/eslint-plugin/lib/rules/no-duplicated-task-keys.js#L84).
```javascript
     "CallExpression[callee.property.name='defineJob'] ObjectExpression BlockStatement": (node) => {
```

When implementing, I didn't know that we had another way than with `defineJob` to create a Job.

Do you have any other example that needs to be fixed with the current implementation?

We can add any other examples on [/packages/eslint-plugin/tests/lib/rules/no-duplicated-task-keys.js:57](https://github.com/triggerdotdev/trigger.dev/blob/main/packages/eslint-plugin/tests/lib/rules/no-duplicated-task-keys.js#L57), and run a `pnpm test` on [/packages/eslint-plugin](https://github.com/triggerdotdev/trigger.dev/blob/main/packages/eslint-plugin), this should show any invalid implementation (and it's easier than create a whole project to test this).

I already have some stuff on my queue, but you can assign this to me.

**Comment by ologbonowiwi** [comment]
Fix on #398. Can you review it when available, @ericallam?

I tested your example and added some more.
